app/src/vanilla/java/cm/aptoide/pt/view/settings/NewAccountPresenter.java

```diff
@@ -0,0 +1,239 @@
+package cm.aptoide.pt.view.settings;
+
+import android.content.SharedPreferences;
+import cm.aptoide.accountmanager.Account;
+import cm.aptoide.accountmanager.AptoideAccountManager;
+import cm.aptoide.pt.account.AccountAnalytics;
+import cm.aptoide.pt.crashreports.CrashReport;
+import cm.aptoide.pt.preferences.managed.ManagedKeys;
+import cm.aptoide.pt.presenter.Presenter;
+import cm.aptoide.pt.presenter.View;
+import rx.Observable;
+import rx.Scheduler;
+
+/**
+ * Created by franciscocalado on 13/03/18.
+ */
+
+public class NewAccountPresenter implements Presenter {
+
+  public static final int EDIT_STORE_REQUEST_CODE = 1230;
```

Comment:
Can be private if there is no necessity of exposing it to outer classes

Reply:
fixed